.pre-commit-config.yaml

```diff
@@ -70,7 +70,7 @@ repos:
         files: deploy/cloudformation/.*yml
 
   - repo: https://github.com/aws-cloudformation/cfn-lint
-    rev: v0.87.2
+    rev: v1.2.5.a8
```

Comment:
See https://github.com/elastic/cloudbeat/pull/2196

Reply:
Oke, shall I just "rollback" this line to the `v0.87.2`? 

I'm doing.